snuba/datasets/storages/errors_common.py

```diff
@@ -144,6 +145,7 @@
     EventIdColumnProcessor(),
     MappingOptimizer("tags", "_tags_hash_map", "events_tags_hash_map_enabled"),
     ArrayJoinKeyValueOptimizer("tags"),
+    UUIDColumnProcessor(set(["event_id"])),
```

Comment:
Why not trace_id as well? I know we added that to the transactions storage but seems like it would work here too?

Reply:
Because the transaction column is not transformed into `replaceAll('transaction_id', .....)`. This happens for event_id through the EventIdColumnProcessor, but that does no transform transaction_id in the query. Not sure that is intentional.

Reply:
If we're not ready to use the promoted column for trace_id yet then I don't think we should add it to the list of prewhere candidates in this PR. We'd get the most benefit when applying those two things together.

Reply:
removed from prewhere